### PR TITLE
PAT-1065 - Create manage soc cases preprod namespace and AWS resources

### PR DESCRIPTION
**Pull request by timharrison-moj**
Creates manage-soc-cases-preprod namespace.
Creates RDS postgresql instance
Creates Redis instance
Creates AWS S3 buckets and permissions

### CODE REVIEW

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/manage-soc-cases-preprod/02-limitrange.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  name: limitrange
+  namespace: manage-soc-cases-preprod
+spec:
+  limits:
+  - default:
+      cpu: 1000m
+      memory: 2048Mi
+    defaultRequest:
+      cpu: 10m
+      memory: 1024Mi
```
Comment (digitalronin) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/manage-soc-cases-preprod/02-limitrange.yaml:13]: Please replace this with the standard default memory request of 100Mi

For more information, please see this page: https://user-guide.cloud-platform.service.justice.gov.uk/documentation/concepts/namespace-limits.html#namespace-container-resource-limits

Comment (timharrison-moj) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/manage-soc-cases-preprod/02-limitrange.yaml]: Thanks for the quick review turnaround. I've updated the request memory as suggested, and added slack-channel and team-name to annotations.

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/manage-soc-cases-preprod/00-namespace.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  name: manage-soc-cases-preprod
+  labels:
+    cloud-platform.justice.gov.uk/is-production: "false"
+    cloud-platform.justice.gov.uk/environment-name: "preprod"
+  annotations:
+    cloud-platform.justice.gov.uk/business-unit: "HMPPS"
+    cloud-platform.justice.gov.uk/application: "manage-soc-cases"
+    cloud-platform.justice.gov.uk/owner: "Digital Prison Services: dps-hmpps@digital.justice.gov.uk"
+    cloud-platform.justice.gov.uk/source-code: "https://github.com/ministryofjustice/manage-soc-cases.git"
+    cloud-platform.justice.gov.uk/slack-channel: "#pathfinder-service"
```
Comment (digitalronin) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/manage-soc-cases-preprod/00-namespace.yaml:13]: Sorry to be a pain, I should have specified that this needs to be without the leading `#` My bad.

Comment (timharrison-moj) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/manage-soc-cases-preprod/00-namespace.yaml]: No problem. That's updated now.

**Pull request merged**